### DISCUSSION TITLE
是否在不同部署的情况下不应显示不可用功能?

### DISCUSSION THREAD

**Discussion by Borber** [lobehub/lobehub#3582] [General Question] [answered]
![image](https://github.com/user-attachments/assets/b82f6791-7c2f-497b-ac78-91381b89c687)

不可用功能但是直接显示在了 pwa 客户端上, 且无法关闭

**Comment by arvinxx** [comment, marked as answer]
应该展示，不然用户可能过了几个月都不知道我们功能上是支持知识库的。只是部署模式不对。

如果需要隐藏的话，可以试下feature Flag =-knowledge_base
